modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -87,11 +87,8 @@ officially supported for mounting `/var` or a subdirectory of `/var`, such as `/
 ====
 For disk sizes larger than 100GB, and especially disk sizes larger than 1TB, create a separate `/var` partition. See "Creating a separate `/var` partition" and this link:https://access.redhat.com/solutions/5587281[Red Hat Knowledgebase article] for more information.
 ====
-+
-[IMPORTANT]
-====
-Kubernetes supports only two file system partitions. If you add more than one partition to the original configuration, Kubernetes cannot monitor all of them.
-====
+
+* Kubernetes supports storing the containers directory on a separate disk. You must have a single filesystem for `/var`, so that you can use a separate partition for `/var/lib/containers`. Kubernetes supports only these two configurations. Kubernetes does not monitor any other type of file system partition.
```

Comment:
I may have suggested this change but I find it a bit confusing..

```suggestion
* Kubernetes supports only two file system partitions.  You can either have a single filesystem for `/var` or you can have a dedicated filesystem for your container runtime.  One could use a separate partition for `/var/lib/containers`.  These are the only two supported ways to have Kubernetes monitor the filesystem.
```

Reply:
Yes, so in this doc, we either support /var/lib/containers is on the same partition as /var.  Or you have a separate partition for /var/lib/containers.

It is also possible to change the location of /var/lib/containers via a crio-storage config change.  I will be working on a post to mention this as its not very well documented.

Reply:
Thanks, @kannon92 .

I updated the content. WDYT?

Reply:
I think there is still a question on this comment.